storage/imx6ull-flashnor/flashnor-qspi.c

```diff
@@ -20,8 +20,11 @@
 #include "flashnor-drv.h"
 
 
+#define min(a, b) ((a) > (b) ? (b) : (a))
```

Comment:
Simple version of this macro is unsafe, see: https://stackoverflow.com/questions/3437404/min-and-max-in-c

Use uppercase names for macros